eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -206,18 +206,6 @@
         "new": "method <V> reactor.core.publisher.Flux<V> reactor.core.publisher.Flux<T>::switchOnFirst(java.util.function.BiFunction<reactor.core.publisher.Signal<? extends T>, reactor.core.publisher.Flux<T>, org.reactivestreams.Publisher<? extends V>>, boolean) @ com.azure.core.util.paging.ContinuablePagedFluxCore<C, T, P extends com.azure.core.util.paging.ContinuablePage<C extends java.lang.Object, T extends java.lang.Object>>",
         "justification": "Reactor added new final APIs to Flux."
       },
```

Comment:
Dont need these anymore?

Reply:
Yeah, I was thinking of going through the RevApi suppressions in another PR to remove what is no longer relavent.